Makefile

```diff
@@ -803,6 +803,9 @@ cache-export-docker-load: $(LINUXKIT)
 ## will skip image if not found in cache
 cache-export-docker-load-all: $(LINUXKIT) $(addsuffix -cache-export-docker-load,$(PKGS_DOCKER_LOAD))
 
+proto-vendor:
+	@$(DOCKER_GO) "cd pkg/pillar ; go mod vendor" $(CURDIR) proto
+
```

Comment:
Should we document this? And why don't we also the libs-vendor and all-vendor targets?

Reply:
I do not know. We got used to this command and find it very convenient, so I just bring it back to live :)

Reply:
It's actually documented in the make help. Seems enough